docs/files/src/param.txt

```diff
@@ -12,15 +12,13 @@ RK4                                 = Euler         # particles advection method
 Xi_min                              = 1.0e-14       # tolerance criterion for the convergence of the non-linear flow, default is 1.0e-14
 random_initial_strain               = 0.0           # non-dimensional value for the initial strain perturbation for the entire domain, default is 0.0 
 pressure_const                      = -1.0          # set constant pressure value for the domain (relevant only when 2D is plain view), default is -1.0 (i.e. not used) [Pa]
-initial_dynamic_range               = False         # method to smoothen convergence of the velocity field in scenarios with wide viscosity range, default is False [True/False]
+initial_dynamic_range               = False         # method to smoothen convergence of the velocity field in scenarios with wide viscosity range, default is False [True/False], Gerya (2019)
 periodic_boundary                   = False         # allows simulation with periodic boundary in the horizontal direction, default is False [True/False]
 high_kappa_in_asthenosphere         = False         # mimics high heat transport in the asthenosphere increasing its thermal diffusivity coefficient, default is False [True/False]
-
-
-basal_heat                          = -1.0          # default is -1.0
+basal_heat                          = -1.0          # set basal heat flux value, default is -1.0 (i.e. not used) [W/m^2]
 
 # Particles options
-particles_per_element               = 1100          # default is 81
+particles_per_element               = 1100          # n. of Lagrangian particles in each element, default is 81
 particles_per_element_x             = 0             # n. of Lagrangian particles in the horizontal direction, default is 0 for automatic calculation
 particles_per_element_z             = 0             # n. of Lagrangian particles in the vertical direction, default is 0 for automatic calculation
 particles_perturb_factor            = 0.0           # default is 0.5 [values are between 0 and 1]
```

Comment:
@victorsacek Can you add a description to this parameter? I think we missed it during the discussion